pkg/meta/repodb/boltdb_wrapper.go

```diff
@@ -13,13 +18,23 @@ import (
 	ispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog"
+	"github.com/sigstore/cosign/pkg/cosign/pkcs11key"
```

Comment:
These should be used in a zot library which is imported in repodb, not part of repodb itself.
This library should be usable in both bold and dynamodb implementation later.
Actually this library should be usable without repoDB.